services/motion/builtin/builtin_test.go

```diff
@@ -458,6 +458,7 @@ func TestMoveOnMap(t *testing.T) {
 }
 
 func TestMoveOnMapTimeout(t *testing.T) {
+	t.Skip()
```

Comment:
Can you make a ticket for reintroducing this test?

Reply:
I'll just properly fix the test here.